cookbooks/bcpc-hadoop/recipes/hdfsdu_pre_deploy.rb

```diff
@@ -0,0 +1,16 @@
+# Setup HDFSDU config
+
+include_recipe 'bcpc-hadoop::hdfsdu_kerberos'
+
+hdfsdu_keytab = ::File.join(node['bcpc']['hadoop']['kerberos']['keytab']['dir'],
+  node[:bcpc]['hadoop']['kerberos']['data']['hdfsdu']['keytab'])
+hdfsdu_hdfs_user = node['hdfsdu']['hdfs_user']
+
+node.override[:hdfsdu][:service_download_url] = get_binary_server_url
+node.override[:bcpc][:hadoop][:hdfs][:dfs][:cluster][:administrators] = \
+  node[:bcpc][:hadoop][:hdfs][:dfs][:cluster][:administrators] + ',' + \
```

Comment:
There should be no need to `override` as you will just add to the existing. This of course depends on the load order, otherwise it will just not exist.